boards/arm/nucleo_wl55jc/doc/nucleo_wl55jc.rst

```diff
@@ -0,0 +1,289 @@
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/nucleo_wl55jc/nucleo_wl55jc_defconfig``
+
+
+Connections and IOs
+===================
+
+Nucleo WL55JC Board has 4 GPIO controllers. These controllers are responsible
+for pin muxing, input/output, pull-up, etc.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+.. rst-class:: rst-columns
+
+- LPUART_1 TX/RX : PA3/PA2 (ST-Link Virtual Port Com)
+
+System Clock
+------------
+
+Nucleo WL55JC System Clock could be driven by internal or external oscillator,
+as well as main PLL clock. By default System clock is driven by HSE clock at
+32MHz.
+
+Serial Port
+-----------
+
+Nucleo WL55JC board has 2 (LP)U(S)ARTs. The Zephyr console output is assigned
+to LPUART_1.
+Default settings are 115200 8N1.
+
+
+Programming and Debugging
+*************************
+
+Applications for the ``nucleo_wl55jc`` board configuration can be built the
+usual way (see :ref:`build_an_application`).
+
+Flashing
+========
+
+Nucleo WL55JC board includes an STLINK-V3 embedded debug tool
+interface.  This interface is supported by the openocd version included in the
+Zephyr SDK since v0.11.0.
+
+
+Flashing an application to Nucleo WL55JC
+----------------------------------------
+
+Connect the Nucleo WL55JC to your host computer using the USB port.
+Then build and flash an application. Here is an example for the
+:ref:`hello_world` application.
+
+Run a serial host program to connect with your Nucleo board:
+
+.. code-block:: console
+
+   $ minicom -D /dev/ttyUSB0
+
+Then build and flash the application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: nucleo_wl55jc
+   :goals: build flash
+
+You should see the following message on the console:
+
+.. code-block:: console
+
+   Hello World! arm
+
+Debugging
+=========
+
+You can debug an application in the usual way.  Here is an example for the
+:ref:`blinky-sample` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/blinky
+   :board: nucleo_wl55jc
+   :maybe-skip-config:
+   :goals: debug
+
+.. _Nucleo WL55JC website:
+   https://www.st.com/en/evaluation-tools/nucleo-wl55jc.html
+
+.. _STM32WL55JC on www.st.com:
+   https://www.st.com/en/microcontrollers-microprocessors/stm32wl55jc.html
+
+.. STM32WL55JC datasheet:
```

Comment:
Can you check the reference here?
This doesn't display as other addresses.

Reply:
Fixed (missing "_" in front of name)